### PR TITLE
doc: tomli is not needed for Python >= 3.11

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -135,8 +135,8 @@ be specified in this file (without the preceding dashes), for example:
 
 Codespell will also check in the current directory for a ``pyproject.toml``
 (or a path can be specified via ``--toml <filename>``) file, and the
-``[tool.codespell]`` entry will be used as long as the tomli_ package
-is installed, for example:
+``[tool.codespell]`` entry will be used, but only if the tomli_ package
+is installed for versions of Python prior to 3.11, for example:
```
Comment (skangas) [inline README.rst:139]: Perhaps "... for versions of Python prior to 3.11. For example:"

Comment (DimitriPapadopoulos) [reply README.rst:139]: It looks better indeed, but I tried to minimally change the existing text.

**Comment by janosh** [comment]
Could this docs example include a line showing how to set `ignore-words-list`?

I tried

```toml
[tool.codespell]
ignore-words-list = "fpr,te,nd"
```

on py 3.11 but seemed to have no effect.

**Comment by DimitriPapadopoulos** [comment]
Someone could try to improve the documentation, but not in this merge request. Open an issue instead.

**Comment by DimitriPapadopoulos** [comment]
@janosh That said, it works for me **if Python supports TOML**:
```
$ cat > FILE
fpr
TE
nd
$ 
$ codespell 
./FILE:1: fpr ==> for, far, fps
./FILE:2: TE ==> THE, BE, WE, TO
./FILE:3: nd ==> and, 2nd
$ 
$ cat > pyproject.toml
[tool.codespell]
ignore-words-list = "fpr,te,nd"
$ 
$ codespell
$ 
```

**Comment by janosh** [comment]
@DimitriPapadopoulos Sorry my mistake. `codespell` was being run by `pre-commit` which I'd falsely assumed was running with py 3.11 but was actually 3.9.

**Comment by DimitriPapadopoulos** [comment]
@larsoner Would you agree to this small documentation change?

**Pull request merged**